au4/src/app/main.cpp

```diff
@@ -145,7 +145,7 @@ int main(int argc, char** argv)
 
     std::vector<char*> argvUTF8;
     for (auto& arg : argvUTF8Q) {
-        argvUTF8.push_back(arg.data());
+        argvUTF8.emplace_back(arg.data());
```

Comment:
The use of emplace_back does not matter here, because only a pointer is inserted (thanks Qt Implicit Sharing), and not some structure by which memory is allocated and copied.